### PR TITLE
Update to latest otelcol core

### PR DESCRIPTION
**Pull request by jpkrohling**
In preparation for v0.64.0

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
Converting to draft, as it should be merged only when core v0.64.0 is released. At the current state, it serves only to confirm that contrib builds with the latest core.

File: cmd/configschema/go.mod [hunk 1]
```diff
@@ -5,11 +5,11 @@ go 1.18
 require (
 	github.com/fatih/structtag v1.2.0
 	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib v0.63.0
+	github.com/open-telemetry/opentelemetry-collector-contrib v0.0.0-00010101000000-000000000000
```
Comment (mx-psi) [inline cmd/configschema/go.mod:8]: I don't think this is needed

**Comment by jpkrohling** [comment]
I'm closing this in favor of the PR created after the release of the core, which includes other changes, such as the changelog.

**Comment by jpkrohling** [comment]
New PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16210

**Pull request closed without merging**